meta/main.yml

```diff
@@ -4,11 +4,14 @@ galaxy_info:
   description: OMERO.web django-prometheus exporter
   company: Open Microscopy Environment
   license: BSD
-  min_ansible_version: 2.3
+  min_ansible_version: 2.10
```

Comment:
do we need that bump?
I think this was needed when declaring dependencies via a requirement file

Reply:
> do we need that bump? I think this was needed when declaring dependencies via a requirement file

I will revert it. See https://github.com/ome/ansible-role-ssl-certificate/pull/19#discussion_r1457360568

Reply:
fixed in https://github.com/ome/ansible-role-omero-web-django-prometheus/pull/8/commits/55cf85494621cfd6aa4557be1b6924d440bc47e9